practise.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Practise</title>
+    <link rel="stylesheet" href="styles.css">
```

Comment:
Don't forget the normalize.css (though I think Jo and Ramsey might have added it)